<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recources.qrc">
    <normaloff>:/psy.png</normaloff>:/psy.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_ok">
        <property name="text">
         <string>Запуск</string>
        </property>
        <property name="icon">
         <iconset resource="recources.qrc">
          <normaloff>:/button_ok.png</normaloff>:/button_ok.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actIconMode"/>
   <addaction name="actListMode"/>
   <addaction name="separator"/>
   <addaction name="actConfig"/>
   <addaction name="actDebug"/>
  </widget>
  <action name="act_exit">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actIconMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/view-list-icons.png</normaloff>:/view-list-icons.png</iconset>
   </property>
   <property name="text">
    <string>ListView</string>
   </property>
  </action>
  <action name="actListMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/view-list-tree.png</normaloff>:/view-list-tree.png</iconset>
   </property>
   <property name="text">
    <string>TableView</string>
   </property>
  </action>
  <action name="actDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/debug-run.png</normaloff>:/debug-run.png</iconset>
   </property>
   <property name="text">
    <string>Отладка</string>
   </property>
  </action>
  <action name="actConfig">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
